### PR TITLE
Remove SiteRecovery psd1 for Az

### PR DESCRIPTION
**Pull request by MiYanni**
## Description

The psd1 for SiteRecovery shouldn't exist for Az as those cmdlets are part of the RecoveryServices module.
This PR also moves StaticAnalysis (Az) into Package with everything else. And, fixes a StaticAnalysis issue.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1 [hunk 1]
```diff
@@ -1,232 +0,0 @@
...
-# CLRVersion = ''
-
-# Processor architecture (None, X86, Amd64) required by this module
-# ProcessorArchitecture = ''
-
-# Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Profile'; ModuleVersion = '0.2.2'; })
-
-# Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = '.\AutoMapper.dll',
```
Comment (markcowl) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1]: Are these assemblies in the Az.RecoveryServices RequiredAssemblies?

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1 [hunk 2]
```diff
@@ -1,232 +0,0 @@
...
-# ScriptsToProcess = @()
-
-# Type files (.ps1xml) to be loaded when importing this module
-# TypesToProcess = @()
-
-# Format files (.ps1xml) to be loaded when importing this module
-# FormatsToProcess = @()
-
-# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
-NestedModules = @('.\Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.dll')
```
Comment (markcowl) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1]: This should be included in Az.RecoveryServices

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1 [hunk 3]
```diff
@@ -1,232 +0,0 @@
...
-
-# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
-FunctionsToExport = @()
-
-# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
-CmdletsToExport = 'Edit-AzRecoveryServicesAsrRecoveryPlan',
```
Comment (markcowl) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1]: These should be included in Az.RecoveryServices

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1 [hunk 4]
```diff
@@ -1,232 +0,0 @@
...
-    'Update-AzRecoveryServicesAsrProtectionDirection', 
-    'Update-AzRecoveryServicesAsrRecoveryPlan', 
-    'Update-AzRecoveryServicesAsrServicesProvider', 
-    'Update-AzRecoveryServicesAsrvCenter'
-
-# Variables to export from this module
-# VariablesToExport = @()
-
-# Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
-AliasesToExport = 'Edit-ASRRP', 'Edit-ASRRecoveryPlan', 'Get-ASRAlertSetting',
```
Comment (markcowl) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1]: same comment

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1 (hunk 1 above)
Comment (MiYanni) [reply src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1]: Added.

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1 (hunk 2 above)
Comment (MiYanni) [reply src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1]: Done.

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1 (hunk 3 above)
Comment (MiYanni) [reply src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1]: Done.

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1 (hunk 4 above)
Comment (MiYanni) [reply src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1]: Done.

**Comment by MiYanni** [comment]
@azuresdkci Retest this please

**Comment by MiYanni** [comment]
@azuresdkci Retest this please

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Az.RecoveryServices.psd1 [hunk 5]
```diff
@@ -54,7 +54,8 @@ DotNetFrameworkVersion = '4.7.2'
 RequiredModules = @(@{ModuleName = 'Az.Profile'; ModuleVersion = '0.3.0'; })
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = '.\Microsoft.Azure.Commands.RecoveryServices.Backup.Models.dll', 
+RequiredAssemblies = '.\AutoMapper.dll',
```
Comment (markcowl) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Az.RecoveryServices.psd1]: Please add this into Az once the appropriate changes have been made

Comment (MiYanni) [reply src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Az.RecoveryServices.psd1]: Will do.

**Pull request merged**